### DISCUSSION TITLE
Why there is no release ?

### DISCUSSION THREAD

**Discussion by Awilum** [clue/framework-x#157] [General]
Project looks interesting, but why there is no published release ? 
Is there any plans for stable release ?

**Comment by SimonFrings** [comment]
Hey @Awilum

This is a good question.  We're still working on some major features (mostly documentation) that we want to see as part of our stable release (e.g. file handling, streams with Framework X etc.). I can't give you an specific date but I can assure you, that we want to achieve the goal of giving this project it's first major release.

For now it's safe to use the development version as we make sure that our code is 100% tested! We're always happy about feedback, let us know if there's anything else you're missing from this project👍
